helm/grafana/values.yaml

```diff
@@ -16,6 +16,64 @@ pgClusterRecoveryTest:
     name: grafana-postgresql-recovery-test
     namespace: monitoring
 
+#-- Grafana postgresql database, using CNPG
+postgresqlCluster:
+  enabled: false
+  name: grafana-postgresql
+  instances: 2
+  storage:
+    size: 2Gi
+  #-- Used for the cilium network policy
+  port: 5432
+  image:
+    # used by renovate
+    # registry: gsoci.azurecr.io
+    registry: gsoci.azurecr.io
+    repository: giantswarm/postgresql-cnpg
+    tag: 17.7
+  objectStorage:
+    s3:
+      #-- For IRSA access
+      serviceAccount:
+        annotations: {}
+      #-- For customers with their own S3 bucket
+      customAccessKeys:
+        use: false
+        endpointURL: ""
+        region: null
+        accessKeyId: ""
+        secretAccessKey: ""
```

Comment:
```suggestion
      # if those values are set, they will be used to access the bucket, otherwise, IRSA will be used
      endpointURL: ""
      region: null
      accessKeyId: ""
      secretAccessKey: ""
```